neqo-transport/tests/server.rs

```diff
@@ -461,6 +464,7 @@ fn new_token_0rtt() {
     assert!(client.tls_info().unwrap().resumed());
 }
 
+#[cfg(not(feature = "fuzzing_t"))]
```

Comment:
Maybe you can move the Retry testing to a new file and just disable that instead.